src/components/account/UsernamePasswordForm.tsx

```diff
@@ -59,40 +61,43 @@ const UsernamePasswordForm: FunctionComponent<UsernamePasswordFormProps> = ({onS
   }
 
   return (
-    <form>
-      <InputField
-        id="username"
-        label="Username"
-        value={username}
-        onChange={e => setUsername(e.target.value)}
-        autoComplete="username"
-        autoFocus={true}
-      />
-      <InputField
-        id="password"
-        label="Password"
-        value={password}
-        type="password"
-        onChange={e => setPassword(e.target.value)}
-        autoComplete="current-password"
-      />
-      <Button
-        fullWidth
-        color="primary"
-        variant="contained"
-        disabled={isLoading || username === '' || password === ''}
-        sx={{mb: 2}}
-        onClick={e => {
-          handleLogin(e)
-        }}>
-        {isLoading ? 'Logging you in...' : 'Sign In'}
-      </Button>
-      {errorMessage && (
-        <Alert variant="outlined" color="error" sx={{mt: 2, alignItems: 'center'}}>
-          {errorMessage}
-        </Alert>
-      )}
-    </form>
+    <>
```

Comment:
The diff isn't very useful here -- the only changes are 1) wrapping the form in an empty tag, so that 2) the "Forgot password?" nav link can be added below the form.